terraform/network.tf

```diff
@@ -1,10 +1,9 @@
 locals {
-  # VNet uses a shared resource group different from the main App Service resource group
-  network_resource_group_name = "RG-CDT-PUB-SHRD-W-P-001"
-}
+  # VNet uses a shared Resource Group, different from App Service Resource Group we use for Benefits stuff
+  network_resource_group_name = local.is_prod ? "RG-CDT-PUB-SHRD-W-P-001" : "RG-CDT-PUB-D-001"
+  vnet_name                   = local.is_prod ? "VNET-CDT-PUB-SHRD-W-P-001" : "VNET-CDT-PUB-D-001"
+  subnet_name                 = local.is_prod ? "SNET-CDT-PUB-CALITP-P-001" : "SN-CDT-PUB-CALITP-${local.env_letter}-001"
```

Comment:
Is the right side of this ternary correct? `"SN-CDT-PUB-CALITP-${local.env_letter}-001"`

Should it be `"SNET-CDT-PUB-CALITP-${local.env_letter}-001"`

Reply:
It matches what @cal-itp/cdt-devsecops named it 🤷

Reply:
As part of https://github.com/cal-itp/benefits/issues/1077, maybe we can raise where we are seeing inconsistencies.